### PR TITLE
Create users sorting

### PR DESCRIPTION
**Pull request by Vgoylo**
Added sorting in controller to users

### CODE REVIEW

File: app/controllers/api/v1/users_controller.rb [hunk 1]
```diff
@@ -6,8 +6,32 @@ class UsersController < ApplicationController
       attr_accessor :user
 
       def index
-        users = User.all
-        render json: users
+
+        if params[:sort]
```
Comment (TehanovAnton) [inline app/controllers/api/v1/users_controller.rb:10]: look my comments in #11

File: app/controllers/api/v1/users_controller.rb [hunk 2]
```diff
@@ -6,8 +6,11 @@ class UsersController < ApplicationController
       attr_accessor :user
 
       def index
-        users = User.all
-        render json: users
+        render json: params[:sort] ? User.order(sort_params.to_h) : Salon.all
```
Comment (dmokreckiy) [inline app/controllers/api/v1/users_controller.rb:9]: Same as here - https://github.com/iTechArt-RubyLab/online-booking-system/pull/9#pullrequestreview-861681897

File: app/models/visit_service.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class VisitService < ApplicationRecord
+  belongs_to :visit
+  belongs_to :servise
```
Comment (dmokreckiy) [inline app/models/visit_service.rb:3]: servi*c*e

**Review by dmokreckiy — APPROVED** [review]
Please fix `service` association and LGTM

**Review by Rub1st — APPROVED** [review]

**Pull request merged**